src/js/navButtons.js

```diff
@@ -20,18 +20,12 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
      * The back and next navigation buttons
      */
     fluid.defaults("gpii.firstDiscovery.navButtons", {
-        gradeNames: ["fluid.viewComponent", "autoInit"],
+        gradeNames: ["fluid.viewComponent", "gpii.firstDiscovery.tooltip", "autoInit"],
```

Comment:
On reading this over again, I'm noticing that we have an "is a" relationship for the tooltip. However, it may be better to define this as a "has a" where the tooltip is added as a subcomponent.